src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -311,7 +310,7 @@ public void testHaplotypeCallerRemoveAltAlleleBasedOnHaptypeScores() throws IOEx
     /*
      * Calculate rough concordance between two vcfs, comparing only the positions, alleles, and the first genotype.
      */
-    private double calculateConcordance( final File actual, final File expected ) {
+    public static double calculateConcordance( final File actual, final File expected ) {
```

Comment:
@droazen I'm moving these to `VariantContextTestUtils` in public, so test files won't depend on each other, but I'm going to do it with the next round of updates if that's alright to avoid needing to do public pr just for this.